src/s3a_the_gutter.tw2

```diff
@@ -1,174 +1,905 @@
+
+::The Gutter - Ron - Pool game start
+“Sounds good to me.” Ron chuckled. “I’m mighty thirsty.” Ron took the first shot without asking and proceeded to sink two striped balls right away. God dammit.
+
+Unlike everything else in life, Ron wasn’t a slouch at pool. Still, he was overconfident and already sauced, so I could tell that the game was mine if I wanted.
+
+[[I botched a couple shots and let Ron win|The Gutter - Ron - Let him win]].
+[[I carefully planned every shot and then laughed at Ron while I sunk the eight ball|The Gutter - Ron - Won too soon]].
+
+::The Gutter - Ron - Pool game early letter wager
+“Ain’t no way, kiddo. That personal love letter is just too dang important to your daddy Ron to give up.”
+
+I knew the letter couldn’t be what he said, but I couldn’t stand hearing this bullshit. I recoiled and shouted back: “You’re not my real dad. You’re a joke and you always smell like a bathroom died. [[We’ll play for drinks.”|The Gutter - Ron - Pool game start]]
+
+::The Gutter - Ron - Let him win
+Ron did an awful dance around the table. Then, he climbed the table and did several humps with the pool cue between his legs. What a fucking piece of shit.
+
+To speed things up and keep conversation at a minimum, I went to the bar to get his drink. I hailed Toe and said:
+
+[[“Strongest thing you’ve got.”|The Gutter - Ron - Strong drink]]
+[[“One beer. Extra piss.”|The Gutter - Ron - Weak drink]]
+
+::The Gutter - Ron - Won too soon
+Ron stared at me dumbfounded. I got a little too jazzed and slapped the pool stick out of his hand while shouting “whose house?!” I did a chicken dance that I’m not proud of, but it felt amazing in the moment.
+
+Ron sulked up to the bar to grab my drink. I shouted after him, “don’t bother!”
+
+[[“Let’s get another round in. Same bet.”|The Gutter - Ron - Pool game start]]
+
+::The Gutter - Ron - Strong drink
+Toe dumped half a beer, filled the can with whiskey, and handed it to me. I delivered the mess to Ron and he took it down in one victory gulp.
+
+“One more round!” he screamed in my face like a hell dad. “And this time, I’ll give you that fucking letter if you win. But if I win, I get to fuck your mom! Oh wait; I can do that whenever. Hahahahaha.”
+
+[[“Let’s play, you shitlord.”|The Gutter - Ron - Pool victory]]
+
+::The Gutter - Ron - Weak drink
+I passed Ron his beer and he chugged it with abandon. It wasn’t fun to watch and then, when he said “let’s go again,” that wasn’t fun to hear. But I knew there had to be a way to trick his dumb ass if I kept at it.
+
+[[“Fine. I’ll do another round.”|The Gutter - Ron - Pool game start]]
+
+::The Gutter - Ron - Pool victory
+Ron was so shitfaced that he couldn’t hold the cue right. I took my time, letting him get even more drunk and exhausted. When I sank the eight ball, Ron collapsed on the table.
+
+I used a pool cue to prop his body up and grab the letter from his pocket. I noticed his keys jangling on his side too.
```

Comment:
Just to rub it in a little more with dear drunk Ron:

> I poked him hard in the tit with a pool cue to prop his body up, and grabbed the letter from his pocket.

Reply:
Mmmm yes, excellent use of "tit".